packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -224,6 +226,7 @@ export class EditorStore {
   // Tabs
   currentEditorState?: EditorState | undefined;
   openedEditorStates: EditorState[] = [];
+  tabHistory: Stack<EditorState>;
```

Comment:
very nice, we used to have this actually, but we removed as it could cause memory leak. Let's remove this for now. For further explanation, see the note in `updateGraphAndApplication`

Reply:
oh no not the most common source of memory leak 😓 understandable, removed!